src/SDKs/_metadata/apimanagement_resource-manager.txt

```diff
@@ -0,0 +1,11 @@
+2020-06-16 18:18:38 UTC
```

Comment:
'src/SDKs/_metadata/' is not empty. This might be caused by outdated build tools. Please follow this guide to update them and make sure the metadata txt file is generated in 'eng/mgmt/mgmtmetadata': https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#updating-build-tools

Reply:
Remove `sdk/apimanagement/Microsoft.Azure.Management.ApiManagement/src/Generated/code-model-v1` as well please